static/js/settings_emoji.js

```diff
@@ -209,10 +218,12 @@ export function set_up() {
                     contentType: false,
                     success() {
                         $("#admin-emoji-status").hide();
+
                         ui_report.success(
                             $t_html({defaultMessage: "Custom emoji added!"}),
                             emoji_status,
                         );
+
```

Comment:
Can you explain the reason for this code formatting ?

Reply:
@aryanshridhar  I usually calls vs code formatting option. May be it is the reason.

Reply:
I will do something for that

Reply:
I don't think we really need that. You can safely remove these.

Reply:
I did not get it. remove what?